modules/xmpp/JingleSessionPC.js

```diff
@@ -2181,4 +2183,23 @@ export default class JingleSessionPC extends JingleSession {
         return `JingleSessionPC[p2p=${this.isP2P},`
                     + `initiator=${this.isInitiator},sid=${this.sid}]`;
     }
+
+    /**
+     * If the A/B test for suspend video is disabled according to the room's
+     * configuration, returns undefined. Otherwise returns a boolean which
+     * indicates whether the suspend video option should be enabled or disabled.
+     */
+    _forceSuspendVideo() {
+        if (!this.room.options.abTesting
+            || !this.room.options.abTesting.enableSuspendVideoTest) {
+            return;
+        }
```

Comment:
Does suspend video only work on chrome? Could other browsers affect whatever is being AB tested?

Reply:
Thinking about this more, I suppose whatever is being used for analysis could filter user agents.

Reply:
I just tested with firefox, and it seems that suspend does NOT work there. Filtering is a bit tricky, because in this case we care about the results on the remote side (i.e. if I enable or disable suspending my video, how does that affect the other participant). I'm trying to find out if we can filter by conferences with 2 chrome participants for example.